FinSearchUnified/Tests/PluginTest.php

```diff
@@ -226,15 +229,39 @@ public function testArticleExportWithCrossSellingCategories($crossSellingCategor
         $this->assertSame($expectedCount, $exportedCount);
     }
 
-    public function testEmptyValueNotAllowedExceptionIsThrownInExport()
+    public function exceptionProvider()
+    {
+        return [
+            'EmptyValueNotAllowedException' => [
+                'exception' =>  new EmptyValueNotAllowedException(),
+            ],
+            'EntityNotFoundException' => [
+                'exception' => new EntityNotFoundException(),
+            ],
+            'Article exists in one of the cross-sell categories configured' => [
+                'exception' => new Exception(),
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider exceptionProvider
+     *
+     * @param $exception
+     */
+    public function testExceptionsAreThrown($exception)
```

Comment:
This can be typehinted. With a typehint you can remove the parameter in the docblock.
```suggestion
    public function testExceptionsAreThrown(Exception $exception)
```